generated/nidmm/session.py

```diff
@@ -562,8 +535,8 @@ def __setattr__(self, key, value):
         object.__setattr__(self, key, value)
 
     def __getitem__(self, key):
-        rep_caps = []
-        raise TypeError("'Session' object does not support indexing. You should use the applicable repeated capabilities container(s): {}".format(', '.join(rep_caps)))
+        rep_cap_help_text = ''
+        raise TypeError("'Session' object does not support indexing." + rep_cap_help_text)
```

Comment:
"does not support indexing" is no longer descriptive. It was for a while back during development when you could do `session[0].foo()`.

Reply:
How is it no longer descriptive. Session does not in fact support indexing. `session[0].foo()` is illegal now. That is why we put this specific error in, so we would be more descriptive.

Reply:
I see. I misunderstood the context in which the error shows up.

I went ahead to see what Python returns in this case in general. Perhaps we should use the same wording.

```
>>> a = 5
>>> a[4]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'int' object is not subscriptable
```

Reply:
I updated the wording to match. Is the wording added when there are repeated capabilities okay?